cves/CVE-2013-0844.yml

```diff
@@ -64,7 +64,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Given a specificly crafted DK4 file, a remote attacker can trigger an array
+  out-of-bounds error causing total shutdown of the service. Additionally complete
+  integrity of the system is lost due to a total loss of system protection with a
+  total disclosure of all system files.
```

Comment:
I do not understand how a total shutdown of the service would cause total disclosure of the service

Reply:
I will clarify this, but simply put in the error stack file paths can be leaked back to the attacker resulting in an integrity violation